modules/user_accounts/ajax/getData.php

```diff
@@ -0,0 +1,141 @@
+<?php
```

Comment:
See [example](https://github.com/aces/Loris/blob/17.1-dev/modules/dicom_archive/php/dicom_archive.class.inc#L183) of `toJSON()` override that will allow you to do what Dave suggested. It seems indeed to be a nicer way to do it here.